targets/targets.json

```diff
@@ -2794,7 +2794,7 @@
     },
 	"EFR32MG12P332F1024GL125": {
         "inherits": ["EFM32"],
-        "extra_labels_add": ["EFR32MG12", "1024K", "SL_RAIL", "SL_CRYPTO"],
+        "extra_labels_add": ["EFR32MG12", "1024K", "SL_RAIL", "SL_CRYPTO", "MBEDTLS_CONFIG_HW_SUPPORT"],
```

Comment:
Why is `MBEDTLS_CONFIG_HW_SUPPORT` added to `extra_labels_add` on this target instead of to `macros` like on other targets?

Reply:
Oops. Fixed.